scipy/stats/unuran/src/AUTHORS

```diff
@@ -0,0 +1,12 @@
+Josef Leydold (leydold@statmath.wu.ac.at)
+Wolfgang Hoermann (hormannw@boun.edu.tr)
+
+Erich Janka
+Guenther Tirler
+Roman Karawatzki 
+
+
+Source code provided by:
+
+Ernst Stadlober
+Renee Touzin
```

Comment:
TBD if we should remove the last name or add a comment 'files xzy of Renee Touzin were removed, see licence.txt'

Reply:
not relevant for this PR, we need to check in the submodule that contains the UNU.RAN code later